packages/mui-lab/src/LoadingButton/LoadingButton.js

```diff
@@ -5,7 +5,9 @@ import { capitalize, unstable_useId as useId } from '@mui/material/utils';
 import { unstable_composeClasses as composeClasses } from '@mui/base';
 import { styled, useThemeProps } from '@mui/material/styles';
 import Button from '@mui/material/Button';
+import ButtonGroupContext from '@mui/material/ButtonGroup/ButtonGroupContext';
```

Comment:
```suggestion
import { ButtonGroupContext } from '@mui/material/ButtonGroup';
```

The import you have is not allowed. We should export the `ButtonGroupContext` if it is not already exported.

Reply:
@mnajdova Why is this not allowed? Because this is done in the same way in the normal button.
https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Button/Button.js#L13

Reply:
This is a different package, in the Button it's a local import.